README.md

```diff
@@ -5,7 +5,9 @@ KolibriOS-NG is a fork of the [KolibriOS](https://kolibrios.org/) operating syst
 KolibriOS is a hobby operating system for x86-compatible computers, its kernel is written entirely in [Fasm](https://flatassembler.net/) assembly language. Based on [MenuetOS](https://www.menuetos.net/), it uses its own standards and is not fully POSIX or UNIX compliant.
 
 ## Special thanks
+
 The KolibriOS-NG team expresses special thanks to the author of the 32-bit __MenuetOS__, [Ville Turjanmaa](https://www.menuetos.net/contact.htm). We also want to note that all __MenuetOS__ copyrights have been preserved.
 
 ## License
-This project licensed under the terms of __GNU GPL 2.0__ license. See [this](kernel/COPYING.TXT) for details.
+
+Contents of this repository licensed under terms of __GNU GPL 2.0__ unless otherwise specified. See [this](COPYING.TXT) file for details
```

Comment:
```suggestion
Contents of this repository are licensed under the terms of __GNU GPL 2.0__ unless otherwise specified. See [this](COPYING.TXT) file for details
```